app/controllers/hunters_controller.rb

```diff
@@ -18,6 +18,7 @@ def index
   # @param :available_improvements [Boolean] Toggles whether or not to include a list of improvements available to
   #   the hunter
   # @see Hunter#available_improvements
+  # Available_gear contains all gear available to the Hunter's current playbook
```

Comment:
Comment leftover.

Reply:
Fixed with commit d7d6308.

Reply:
I'm still seeing this in the final file chantges.

Reply:
should be fixed as of 59ce133.